webapp/Connector/src/view/Grid.js

```diff
@@ -336,7 +375,7 @@ Ext.define('Connector.view.Grid', {
         var target;
 
         if (Ext.isString(columnName)) {
-            var fields = this.getModel().get('metadata').metaData.fields;
+            var fields = this.getModel().getActiveSheetMetadata().metaData.fields;
```

Comment:
There are some places now that use `this.getModel().get('metadata')` and some that use `this.getModel().getActiveSheetMetadata()`. How can one tell which one they should use?

Reply:
I've changed metadata to metadatas to indicate it's all metadatas, as opposed to the metadata for the currently viewed sheet.